S7.Net/Types/Class.cs

```diff
@@ -340,5 +342,13 @@ public static byte[] ToBytes(object sourceClass)
             }
             return bytes;
         }
+
+        private static double IncreaseToEvenNumber(double numBytes)
+        {
+            numBytes = Math.Ceiling(numBytes);
+            if ((numBytes / 2 - Math.Floor(numBytes / 2.0)) > 0)
+                numBytes++;
```

Comment:
I'd say `if (numBytes & 1 == 1) numBytes++;`

Reply:
I agree, this looks a little bit more clean, though it misses braces around the & operation. But you still would need to do the `Math.Ceiling(numBytes)` and cast the result to `int` to apply the logic operator. Thereafter it is cast back to double for the calling environment. I made a quick test with 10E7 runs and a ranoum value, both approaches needed 3.5 to 4 seconds on my machine. So from the performance side, there is not really a difference.

But I'm happy to change this if you finally merge it into master, because we are forced to use our own fork for almost a year now and cannot participate in new features without merging constantly from master for ourselves.

Reply:
For the sake of readability I'd like a change, doesn't have to be my suggested change, but the current line isn't very expressive with regards to an odd/even check.

About the release part, I'm really behind on that, but my intent is to release soon, would be good to have this included.